prow/config/config.go

```diff
@@ -732,6 +743,18 @@ func parseProwConfig(c *Config) error {
 		c.Gerrit.RateLimit = 5
 	}
 
+	if len(c.GithubReporter.JobTypesToReport) == 0 {
+		c.GithubReporter.JobTypesToReport = append(c.GithubReporter.JobTypesToReport, prowapi.PresubmitJob)
```

Comment:
Is this default we always want? If not I would suggest adding a TODO to update this so we don't forget

Reply:
will switch to presubmit + postsubmit, added a TODO